rules/framework.bzl

```diff
@@ -788,6 +789,13 @@ def _bundle_dynamic_framework(ctx, is_extension_safe, avoid_deps):
 def _bundle_static_framework(ctx, is_extension_safe, outputs):
     """Returns bundle info for a static framework commonly used intra-build"""
     infoplist = _merge_root_infoplists(ctx)
+    plist_out = infoplist
+    framework_dir = "%s/%s.%s" % (ctx.attr.name, ctx.attr.framework_name, ctx.attr.bundle_extension)
+    plist_out = [paths.join(
+        framework_dir,
+        "Info.plist",
+    )]
+    plist_out = _framework_packaging(ctx, "plist", [infoplist], plist_out, None)
```

Comment:
Is it possible (or does it make sense) to move this within `_get_framework_files` like the rest of the "packaging" related things utilizing `_framework_packaging`?

Reply:
Agree - the change @mattrobmattrob mentioned might mitigate this change too:
https://github.com/bazel-ios/rules_ios/pull/607/files#diff-b5ab983bf150dfa86b7e361224e1c9b0b9856b0b57b5893dbc088c6ca6d8ba52L162

Concretely when `apple.virtualize_frameworks` is set - the rules shouldn't generate any actions for the frameworks like `modulemap` or `plist`

Reply:
I moved this to the `_get_framework_files ` but I also had to move the plist generation out of `_bundle_static_framework` so that it could be packaged inside this method. The generated plist is then passed to the `_bundle_static_framework` through `outputs` so that it can be included in the `AppleBundleInfo` provider.